config/manager/manager.yaml

```diff
@@ -30,7 +30,7 @@ spec:
             capabilities:
               drop: ["ALL"]
           command:
-            - /manager
+            - /node-feature-discovery-operator
```

Comment:
nit: this is another unrelated change

Reply:
https://github.com/kubernetes-sigs/node-feature-discovery-operator/pull/57